### PR TITLE
fixes bug 1186482 - deprecate /explosive/

### PR DESCRIPTION
**Pull request by peterbe**

### CODE REVIEW

**Comment by AdrianGaudebert** [comment]
There are many errors in the tests, did you perhaps forget to remove a bunch of things?

**Comment by peterbe** [comment]
@AdrianGaudebert it passed!

**Comment by peterbe** [comment]
@AdrianGaudebert This blocks my work on switching to pipeline. Just saying :)

File: scripts/database/create_heroku_stage_database.sh [hunk 1]
```diff
@@ -1,13 +1,13 @@
 #!/usr/bin/env bash
```
Comment (AdrianGaudebert) [inline scripts/database/create_heroku_stage_database.sh]: Out of scope, but I bet we can actually remove this file.

**Comment by AdrianGaudebert** [comment]
:sunny: :heart_eyes: r+ would review again

**Comment by peterbe** [comment]
I've just rebased it but I think I'm going to merge it after the release. That way we can let it explode (haha!) on stage for a bit first.

**Pull request merged**